.gitignore

```diff
@@ -9,6 +9,8 @@ __pycache__
 *.egg-info
 *.obj
 *.exe.manifest
+*.h5
+*.hdf5
```

Comment:
I don't think adding these excludes is necessary or advisable. While we generally don't want to add binary files to the repo, there are a handful of these in the `test/data` directory, and it's annoying to work around having these files in the exclude, when the best practice to prevent accidentally adding these files is to never run `git add .`.

Reply:
Replacing this by 
```
samples/**/*.h[5df]
```
as hdf files in `samples` are the actual issue.